build/FindQt5.cmake

```diff
@@ -36,6 +36,10 @@ if (WIN32)
       )
 endif(WIN32)
 
+find_package(Qt5Core 5.15.0 QUIET)
```

Comment:
It is not just 5.15.0, but also 5.15.1 and 5.15.2 that doesn't work

Reply:
`find_package(Qt5Core 5.15.0 QUIET)` matches any version starting at 5.15.0. To match only 5.15.0 you'd need `find_package(Qt5Core 5.15.0 EXACT QUIET)`.

Reply:
Ah, I see